### DISCUSSION TITLE
[Question]: Frequent restart of application

### DISCUSSION THREAD

**Discussion by ss-gonda** [danny-avila/LibreChat#4113] [Q&A]
### What is your question?

In api/server/index.js file, we have at the end process.exit for any unhandled exceptions. 
We have rolled it out to a larger audience (150+ active users). Now we are seeing that lot of places the process gets connectivity issues (mostly with Assistants) and in those cases the server is restarted.
Is it possible that we remove this process.exit from the file to avoid this ? Because the system is not interrupted for users and they still get the response and answers but somehow the restarts are happening too frequent. 

### More Details

The deployment infra is on GCP

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
It would be helpful if you share what is causing backend crashes. I help someone manage an instance where they serve to 2000+ users and almost exclusively use Assistants without these issues.

> Is it possible that we remove this process.exit from the file to avoid this ?

This is your prerogative

**Comment by ss-gonda** [comment]
Hi Danny,

We upgraded to .7.5rc2 and looks like the restarts are not coming. Also, looks like it was due to some deployment issues and was only triggering on GCP. Will stick with the base code only.
Thanks for the update.
